jsapp/js/account/security/accessLogs/accessLogs.query.ts

```diff
@@ -50,3 +50,23 @@ export default function useAccessLogsQuery(
     refetchOnWindowFocus: true,
   });
 }
+
+/**
+ * Starts the exporting process of the access logs.
+ * @returns {Promise<void>} A promise that starts the export.
+ */
+export const startAccessLogsExport = (): Promise<void> =>
+  new Promise<void>((resolve, reject) => {
+    fetchPost(endpoints.ACCESS_LOGS_EXPORT_URL, {})
```

Comment:
I see you're handling errors here, so you should also pass options `{notifyAboutError: false}` to not have duplicate notifications :)